src/components/Routes.js

```diff
@@ -28,6 +30,7 @@ const Routes = () => (
     <Route exact path="/articles/create" component={CreateArticles} />
     <Route exact path="/privacy" component={Privacy} />
     <Route exact path="/signup" component={Signup} />
+    <Route path="/updatearticle" component={UpdateArticles} />
```

Comment:
what do you think about `article/update` or `article/edit`?

Reply:
Changes pushed